typescript/package.json

```diff
@@ -135,5 +135,8 @@
     },
     "workspaces": [
         "packages/*"
-    ]
+    ],
+    "dependencies": {
+        "math.gl": "^4.1.0"
```

Comment:
This should probably be in `packages/subsurface-viewer/package.json` instead of here